plugins/edsm.py

```diff
@@ -528,7 +529,13 @@ def cmdr_data(data: CAPIData, is_beta: bool) -> None:
         this.system_link.update_idletasks()
 
 
+TARGET_URL = 'https://www.edsm.net/api-journal-v1'
+if 'edsm' in debug_senders:
+    TARGET_URL = f'http://{DEBUG_WEBSERVER_HOST}:{DEBUG_WEBSERVER_PORT}/edsm'
+
 # Worker thread
+
+
 def worker() -> None:
```

Comment:
Might as well # noqa this.

At some point in the future we'll get around to grepping for all such and see about refactors.  Reduce the noise for now.